[{
	"raw": {},
	"required": [
		"Filename",
		"Size"
	],
	"package": "python3-pytest-rerunfailures",
	"source": "pytest-rerunfailures",
	"version": "12.0-1",
	"section": "universe/python",
	"priority": "optional",
	"architecture": "all",
	"depends": [
		"python3-importlib-metadata | python3 (>> 3.8)",
		"python3-packaging",
		"python3-pytest (>= 6.2)",
		"python3:any"
	],
	"installedSize": 60,
	"maintainer": "Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>",
	"description": "pytest plugin to re-run tests to eliminate flaky failures",
	"homepage": "https://github.com/pytest-dev/pytest-rerunfailures",
	"filename": "pool/universe/p/pytest-rerunfailures/python3-pytest-rerunfailures_12.0-1_all.deb",
	"size": 14052,
	"md5": "ecc8b3a610e64e714ac34b068713c53b",
	"sha1": "ab53cd8e88ff3048a6af03e307d5093a62f075fa",
	"sha256": "924aabb794fb6a57c3dfa006803ac230be46140b21e34a345bd431fd0758f34d",
	"sha512": "fe257f97b2e196c888aadd0faec64a83b3add7141da342b76c662a81bd1a682501627e5999448cad0b7ef604d9be2c75f016ded19a35d0c3ae7b3f0c7ec0bcbd",
	"descriptionMd5": "b39ddd4149884d1e62c3622245de3f54"
}]